library.json

```diff
@@ -1,19 +1,24 @@
 {
   "title": "Timeline",
   "machineName": "H5P.Timeline",
-  "majorVersion": 1,
+  "majorVersion": 2,
```

Comment:
😬  is 1000 a better number, or should we call this NDLATimeline?

Reply:
NDLATimeline for sure!